rucio-server/values.yaml

```diff
@@ -113,6 +113,17 @@ metricsExporterResources:
    cpu: 20m
    memory: 20Mi
 
+ftsRenewal:
+  enabled: 0
+  schedule: "12 */6 * * *"
+  image:
+    repository: rucio/fts-cron
+    tag: latest
+    pullPolicy: Always
+  vo: "cms"
```

Comment:
Maybe take this opportunity to make dummy defaults, not the CMS ones?

Reply:
They are disabled anyway. Feels more like an example then real defaults. I would rather remove everything except the "enabled: 0" in the "values" file.